{"uid":"d91267db2c277af8","name":"User is not allowed to enter alpha characters","fullName":"tests.test_login.TestLogin#test_invalid_input_number","historyId":"2134688c9ef20ecac430c10444678427","time":{"start":1684826921705,"stop":1684826934551,"duration":12846},"description":"User is not allowed to enter alpha characters","descriptionHtml":"<p>User is not allowed to enter alpha characters</p>\n","status":"passed","flaky":false,"newFailed":false,"newBroken":false,"newPassed":true,"retriesCount":2,"retriesStatusChange":true,"beforeStages":[{"name":"setup","time":{"start":1684826874788,"stop":1684826921699,"duration":46911},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false}],"testStage":{"description":"User is not allowed to enter alpha characters","status":"passed","steps":[{"name":"User clicks login icon in nav bar","time":{"start":1684826921706,"stop":1684826927151,"duration":5445},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"Fill phone number","time":{"start":1684826927151,"stop":1684826934468,"duration":7317},"status":"passed","steps":[],"attachments":[],"parameters":[{"name":"phone_number","value":"'aaaa'"}],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":true}],"attachments":[{"uid":"399dc19209f90aa5","name":"stdout","source":"399dc19209f90aa5.txt","type":"text/plain","size":54}],"parameters":[],"stepsCount":2,"attachmentsCount":1,"shouldDisplayMessage":false,"hasContent":true},"afterStages":[{"name":"setup::0","time":{"start":1684826934552,"stop":1684826935035,"duration":483},"status":"passed","steps":[],"attachments":[{"uid":"5cc09c50adf7500d","name":"Screenshot","source":"5cc09c50adf7500d.png","type":"image/png","size":123477}],"parameters":[],"stepsCount":0,"attachmentsCount":1,"shouldDisplayMessage":false,"hasContent":true}],"labels":[{"name":"tag","value":"chrome"},{"name":"story","value":"Login functionality"},{"name":"tag","value":"@pytest.mark.usefixtures('setup')"},{"name":"parentSuite","value":"tests"},{"name":"suite","value":"test_login"},{"name":"subSuite","value":"TestLogin"},{"name":"host","value":"e7a5f6497e16"},{"name":"thread","value":"35-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.test_login"},{"name":"resultFormat","value":"allure2"}],"parameters":[{"name":"browser","value":"'chrome'"}],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[{"uid":"7098a4f678ee356b","status":"broken","statusDetails":"urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='selenium-hub', port=4444): Max retries exceeded with url: /session (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0xffffa89abcd0>: Failed to establish a new connection: [Errno 111] Connection refused'))","time":{"start":1684826614050,"stop":1684826614050,"duration":0}},{"uid":"d4586b93809fb0d4","status":"broken","statusDetails":"urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='selenium-hub', port=4444): Max retries exceeded with url: /session (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0xffff91be2290>: Failed to establish a new connection: [Errno 111] Connection refused'))","time":{"start":1684826531272,"stop":1684826531272,"duration":0}}],"categories":[],"history":{"statistic":{"failed":0,"broken":2,"skipped":0,"passed":1,"unknown":0,"total":3},"items":[{"uid":"f040e2f2341c5e8d","reportUrl":"../4/index.html#testresult/f040e2f2341c5e8d","status":"broken","statusDetails":"urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='selenium-hub', port=4444): Max retries exceeded with url: /session (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0xffffa89abcd0>: Failed to establish a new connection: [Errno 111] Connection refused'))","time":{"start":1684826614050,"stop":1684826614050,"duration":0}},{"uid":"8b39e24d4af6846d","reportUrl":"../3/index.html#testresult/8b39e24d4af6846d","status":"broken","statusDetails":"urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='selenium-hub', port=4444): Max retries exceeded with url: /session (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0xffff91be2290>: Failed to establish a new connection: [Errno 111] Connection refused'))","time":{"start":1684826531272,"stop":1684826531272,"duration":0}}]},"tags":["chrome","@pytest.mark.usefixtures('setup')"]},"source":"d91267db2c277af8.json","parameterValues":["'chrome'"]}